alarm/utils.py

```diff
@@ -0,0 +1,62 @@
+from alarm.binance_utils import extract_binance_data_from_socket, parse_binance_data
+from alarm.models import Candle, Coin
+
+
+def check_if_call_needed(prices):
+    threshold = prices['threshold']
+    last_candle_high_price = prices['last_candle_high_price']
+    current_candle_high_price = prices['current_candle_high_price']
+    last_candle_low_price = prices['last_candle_low_price']
+    current_candle_low_price = prices['current_candle_low_price']
```

Comment:
Instead of "prices" should be "coin_prices_data"

Reply:
++